app/screens/error-screen/error-screen.tsx

```diff
@@ -59,7 +60,35 @@ const styles = EStyleSheet.create({
   },
 })
 export const ErrorScreen = ({ error, resetError }) => {
+  const { logout } = useLogout()
+  const [logoutShown, setLogoutShown] = React.useState(false)
   useEffect(() => crashlytics().recordError(error), [error])
+
+  const logoutAction = async () => {
+    try {
+      await logout()
+      resetError()
+      Alert.alert(translate("common.reauth"), "", [
+        {
+          text: translate("common.ok"),
+          onPress: () => console.log("OK pressed"),
+        },
+      ])
+    } catch (err) {
+      // TODO: figure out why ListItem onPress is swallowing errors
```

Comment:
Not sure what this TODO means.  Can you give more detail?

Reply:
That's a fair point. I took a stab at implementing this as an Apollo Link from `app/app.tsx`, but the problem I was running into was that I couldn't use the `useLogout` hook from outside of a component. So, that's when I tried it this way from `error-screen`.

One thing I noticed after submitting this PR was https://github.com/GaloyMoney/galoy-mobile/blob/main/app/components/global-error/global-error.tsx. It looks like there was code that handled 401 status codes at some point in there, but it doesn't seem to get called from any place anymore. Maybe it would make more sense to get this code working again?

Reply:
> Not sure what this TODO means. Can you give more detail?

This was copy-pasted from https://github.com/GaloyMoney/galoy-mobile/blob/main/app/screens/settings-screen/settings-screen.tsx#L45 and doesn't belong here. If we continue down this path, I will remove.